### PR TITLE
Test Script

### PR DESCRIPTION
**Pull request by MichaelCherry**
Added a test script 'run-tests.sh' to automatically run './manage.py tests' without having to start docker, enter the container, then run the command. Make sure docker is down before running.
Can take a minute, since it first runs 'docker-compose build'

### CODE REVIEW

**Comment by MichaelCherry** [comment]
Now runs tests for backend and frontend; exit for frontend is a little messy (need to click Ctrl+C a few times), but all tests run. May just be missing a simple command for that.

**Comment by caleybrock** [comment]
@MichaelCherry can you attach a screenshot of test output since we don't have it auto-running on PRs yet?

**Comment by MichaelCherry** [comment]
I will add it momentarily. Was trying something to exit watch mode from npm automatically

**Comment by MichaelCherry** [comment]
Here is the output: note you will need to press 'q' after npm test finishes
testoutput

**Comment by MichaelCherry** [comment]
hit "close and comment" by mistake

File: run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Build docker (can comment out if already built)
+docker-compose build
+# Test the frontend
```
Comment (caleybrock) [inline run-tests.sh]: This is great!! 

I find it a little hard to read in the console, so I recommend adding some helpful print statements.
Here's an idea that uses colour, screenshot following.
```
# Constants to add colour to text
BLUE='\033[0;34m'
NC='\033[0m' # No Color

# Test the backened
printf "**** ${BLUE}Begin backend tests${NC} ***\n"
docker-compose run \
  --rm backend ./manage.py test
printf "**** Complete backend tests ***\n"

# Test the frontend
printf "**** ${BLUE}Begin frontend tests${NC} ***\n"
docker-compose run \
  --rm frontend npm test
printf "**** Complete frontend tests ***\n"
```

image

**Comment by caleybrock** [comment]
I don't mind the fact that it continues to watch the frontend - makes iteration a little faster and it isn't a big deal to quit the console.

File: run-tests.sh (hunk 1 above)
Comment (MichaelCherry) [reply run-tests.sh]: I can add this easily enough

Comment (MichaelCherry) [reply run-tests.sh]: Added in latest commit

**Comment by MichaelCherry** [comment]
added --watch to command on frontend; need to click "Ctrl+C" to exit; but does not clear terminal (all tests can be seen now.
image

**Comment by fnormand01** [comment]
I tested it locally. Here are the results:

**From GitBash:**
image

**From Powershell:**
image

Then it opens a GitBash window with the following message:
image

However, if I use this command:
image

I get:
image

Then the screen refreshes and cleans everything and I get:
image


But, as I can see, if it works with Powershell on your machine, the problem I get seems to come from my machine configurations or something on my machine, not the code. That's not a big issue for me, since it's working with GitBash. That being said, your changes are good and it works perfectly..

**Pull request merged**